ledger/src/shred.rs

```diff
@@ -287,6 +287,10 @@ impl ErasureSetId {
     pub(crate) fn store_key(&self) -> (Slot, /*fec_set_index:*/ u64) {
         (self.0, u64::from(self.1))
     }
+
+    pub(crate) fn key(&self) -> (Slot, /*fec_set_index:*/ u32) {
+        (self.0, self.1)
+    }
```

Comment:
oh, this `key` vs `store_key` is pretty bad and confusing.
Can you please leave a comment where one should use each?

Maybe we should remove the `(_, u64)` version, and do the type cast only at the call-sites which use that.

Reply:
yeah you're right i'll remove/deprecate the old `store_key` and only use the `u32` version.

Reply:
for posterity this is being fixed here: https://github.com/solana-labs/solana/pull/34268